radix-transactions/src/model/v2/child_subintent_hashes_v2.rs

```diff
@@ -66,6 +66,63 @@ impl From<ManifestNamedIntent> for ManifestNamedIntentIndex {
     }
 }
 
+//========
+// resolution
+//========
+
+/// This is for use with the [`ResolvableManifestNamedIntent`] trait.
+/// Implementers should panic if a bucket cannot be found.
+pub trait NamedManifestIntentResolver {
```

Comment:
We have pretty much moved all of the resolvable traits to `radix-common` except for this one. Why?

Reply:
I'd like us to align on this first before we merge.

Reply:
Because `ManifestIntent` doesn't exist in `radix-common`... because it's not actually a custom type in the  manifest value model - because we didn't want to open that can of worms at Cuttlefish.